src/Client.cpp

```diff
@@ -109,12 +109,17 @@ void UrsaClient::recv_res(zmq::socket_t &socket) {
         this->command_active = false;
         auto res = json::parse(res_str);
 
-        if (!this->quiet_mode) {
-            if (res["type"] == "error") {
-                spdlog::error(res["error"]["message"].get<std::string>());
-            } else {
-                std::cout << res.dump(4) << std::endl;
+        if (this->force_json) {
+            std::cout << res.dump(4) << std::endl;
+            return;
+        }
+
+        if (res["type"] == "select") {
+            for (auto &res : res["result"]["files"]) {
```

Comment:
it's correct, but maybe change the variable name to avoid shadowing :thinking: (for example to `file`)

Reply:
:thinking: didn't even realized previously, fixed